packages/eas-cli/src/credentials/android/actions/new/SetupBuildCredentials.ts

```diff
@@ -25,6 +29,13 @@ export class SetupBuildCredentials {
 
   async runAsync(ctx: Context): Promise<AndroidAppBuildCredentialsFragment> {
     const { app, name: maybeName } = this.options;
+
+    // copy legacy credentials if user is new to EAS and has legacy credentials
+    const canCopyLegacyCredentials = await canCopyLegacyCredentialsAsync(ctx, app);
+    if (canCopyLegacyCredentials) {
+      await promptUserAndCopyLegacyCredentialsAsync(ctx, app);
+    }
```

Comment:
```suggestion
    if (!ctx.nonInteractive) {
      const canCopyLegacyCredentials = await canCopyLegacyCredentialsAsync(ctx, app);
      if (canCopyLegacyCredentials) {
        await promptUserAndCopyLegacyCredentialsAsync(ctx, app);
      }
    }
```

Reply:
I'd add this `if` because of the assert in `promptUser...Async` (https://github.com/expo/eas-cli/pull/445/files#diff-325afe7218dac2c9caa65cbae656b012e1095fb906d8382b8a1e15d6cbfc0f97R37)